lilypad/lilypad-v2-testnet/quick-start/setting-up-metamask.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Connecting to Lilypad v2 Aurora Testnet
+---
+
+# Setting up metamask
+
+## Setting up Metamask
+
+* Install metamask Extension [here](https://metamask.io/)
+* Next, add the Lilypad Testnet chain to metamask.
+
+{% hint style="info" %}
+Network name: `Lilypad v2 Aurora testnet`
+
+New RPC URL: `http://testnet.lilypad.tech:8545`
+
+Chain ID: `1337`
+
+Currency symbol: `ETH`
+
+Block explorer URL: (leave blank)
+{% endhint %}
+
+To do this, open metamask then click on the network button at the top left of the popup (in the menu bar):
+
+## ![](<../../.gitbook/assets/image (3) (1) (1) (1) (1) (1).png>) \`\` ![](<../../.gitbook/assets/image (5) (1) (1) (1) (1) (1).png>)
+
+Then click the "Add Network" Button.
+
+<figure><img src="../../.gitbook/assets/image (15) (1) (1) (1).png" alt=""><figcaption></figcaption></figure>
+
+Next, click on "Add a network manually" at the bottom of the page and enter the Lilypad Testnet details:
+
+<figure><img src="../../.gitbook/assets/mm_five.png" alt=""><figcaption></figcaption></figure>
+
+Click Save. :tada:
```

Comment:
Let's just make sure <code>:tada:</code> translates to 🎉- if not, we can use emojis directly.